kubevirt_controller/datadog_checks/kubevirt_controller/metrics.py

```diff
@@ -0,0 +1,29 @@
+# (C) Datadog, Inc. 2024-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+METRICS_MAP = {
+    "kubevirt_virt_controller_leading_status": "virt_controller.leading_status",
+    "kubevirt_virt_controller_ready_status": "virt_controller.ready_status",
+    "kubevirt_vm_error_status_last_transition_timestamp_seconds": "vm.error_status_last_transition_timestamp_seconds",
+    "kubevirt_vm_migrating_status_last_transition_timestamp_seconds": "vm.migrating_status_last_transition_timestamp_seconds",  # noqa: E501
+    "kubevirt_vm_non_running_status_last_transition_timestamp_seconds": "vm.non_running_status_last_transition_timestamp_seconds",  # noqa: E501
+    "kubevirt_vm_running_status_last_transition_timestamp_seconds": "vm.running_status_last_transition_timestamp_seconds",  # noqa: E501
+    "kubevirt_vm_starting_status_last_transition_timestamp_seconds": "vm.starting_status_last_transition_timestamp_seconds",  # noqa: E501
```

Comment:
these can be formatted similar to this https://github.com/DataDog/integrations-core/blob/master/envoy/datadog_checks/envoy/metrics.py#L32-L34 to avoid the noqa tag